Frameworks/Foundation/NSProxy.mm

```diff
@@ -202,12 +210,40 @@ - (BOOL)conformsToProtocol:(Protocol*)aProtocol {
     return class_conformsToProtocol(forwardClass, aProtocol);
 }
 
+// NOTE: long return value to allow nonfatal continuation to get a "valid" result (for non-fpret/non-stret calls)
+static long _throwUnrecognizedSelectorException(id self, Class isa, SEL sel) {
```

Comment:
I'd prefer we expose it in NSObject as an internal function and use it from there, but I'm won't be horribly hurt if we don't.